VetClinic.BLL/Services/Realizations/UserService.cs

```diff
@@ -56,7 +56,7 @@ public async Task<bool> UpdateUser(string id, User inputUser, IEnumerable<Identi
                 user.LastName = inputUser.LastName;
                 user.Email = inputUser.Email;
                 user.PhoneNumber = inputUser.PhoneNumber;
-
+                
```

Comment:
Please align code

Reply:
Isn't it already aligned? It does have one tab after if block brace